tools/project_manager/external_catalog_search.php

```diff
@@ -200,6 +208,8 @@ function do_search_and_show_hits()
             echo   "<td class='left-align top-align'>$value</td>";
             echo "</tr>\n";
         }
+        // For adding test cases
```

Comment:
Leftover debugging?

Reply:
Yeah, deliberately, since it's slightly non-obvious. But I can remove it.

Reply:
Perhaps instead add a comment in `MARCRecordTest.php` about how to generate the encoded data, eg:
```php
// To generate data for these test functions, use:
// base64_encode(serialize($rec)) where $rec is the output of yaz_record()
```